flyteplugins/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go

```diff
@@ -60,6 +60,14 @@ func GetExecutionEnvVars(id pluginsCore.TaskExecutionID, consoleURL string) []v1
 			Name:  "FLYTE_INTERNAL_EXECUTION_DOMAIN",
 			Value: nodeExecutionID.Domain,
 		},
+		{
+			Name: "FLYTE_INTERNAL_POD_NAME",
```

Comment:
Let's rename this to `_F_PN` and leave a comment about what it points to.

Reply:
Done